administration/src/cards/Card.ts

```diff
@@ -65,11 +65,22 @@ export const getExtensions = ({ extensions }: Card): ExtensionWithState[] => {
 export const hasInfiniteLifetime = (card: Card): boolean =>
   getExtensions(card).some(({ extension, state }) => extension.causesInfiniteLifetime(state))
 
-export const isFullNameValid = ({ fullName }: Card): boolean => {
+const containsNameSpecialCharacters = (fullName: string): boolean =>
+  /[`!@#$%^&*()_+\-=\]{};':"\\|,.<>?~0123456789]/.test(fullName)
+
+const hasValidNameLength = (fullName: string): boolean => {
   const encodedName = new TextEncoder().encode(fullName)
   return fullName.length > 0 && encodedName.length <= MAX_ENCODED_NAME_LENGTH && fullName.length <= MAX_NAME_LENGTH
 }
 
+const hasNameAndForename = (fullName: string): boolean => {
+  const names = fullName.split(' ')
+  return names.length > 1 && names.every(name => name.length > 1)
```

Comment:
:wrench: This leads to failing checks if a user accidentally puts two spaces

Reply:
i added a separate messages that multiple spaces between names are not allowed. i think just stripping them is difficult since the name on the card should be the same than entered in the form field

Reply:
I see. How about solving this in a programmatic way when actually creating the card by just stripping multiple spaces then? I guess I'd prefer that over having a separate check with error messages here